app/views/replies/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="box wide">
+  <div class="legend right">
+    <%= link_to_filter('unread') %> |
+    <%= link_to_filter('all') %> |
+    <%= link_to_filter('comments') %> |
+    <%= link_to_filter('stories') %>
+
+  </div>
+
+  <div class="legend">
+    <%= @heading %>
+  </div>
+</div>
+
+<ol class="comments comments1">
+  <% @replies.each do |reply| %>
+  <li class="comments_subtree">
```

Comment:
Please indent this `<li>` inside the `each` block, and the contents of the `if` blocks below.